extension/src/paymentHandler/manual-capture.handler.js

```diff
@@ -50,4 +53,4 @@ async function execute(paymentObject) {
   }
 }
 
-module.exports = { execute }
+export default execute
```

Comment:
What is the reason sometimes you use `export default { execute }` and sometimes `export default execute`?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/929/commits/806c44139f4d4c6c6b08ca39a8752a26f2b5e0ba